js/views/aliases-list.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * ownCloud - Mail
+ *
+ * This file is licensed under the Affero General Public License version 3 or
+ * later. See the COPYING file.
+ *
+ * @author Tahaa Karim <tahaalibra@gmail.com>
+ * @copyright Tahaa Karim 2016
+ */
+
+define(function(require) {
```

Comment:
so you're using a CompositeView and render the collection manually in the Handlebars template. If you instead use a CollectionView for the aliases an pass the aliases as 'collection' constructor option you'll get an auto-updating view that adds/removes/updates any changes you do on the collection/models.